src/types/timelineTypes.ts

```diff
@@ -0,0 +1,54 @@
+import {VNode} from 'preact';
+
+export interface DefaultPreviewProps {
+  virtualTime: number;
+}
+
+export interface PreviewProps {
+  defaultPreviewProps: DefaultPreviewProps;
+}
+
+export interface ThumbnailInfo {
+  height: number;
+  url: string;
+  width: number;
+  x: number;
+  y: number;
+}
+
+export interface TimelineMarkerProps {
+  class: string;
```

Comment:
what `class` property means here?

Reply:
not sure actually.. probably can be removed- it is not used anywhere..